website.i2pd.i2pd.json

```
@@ -0,0 +1,105 @@
+                    "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+                }
+            ]
+        },
+        {
+            "name": "openssl",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./config --prefix=/app --openssldir=/app/ssl shared zlib",
+                "make -j $FLATPAK_BUILDER_N_JOBS",
+                "make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.openssl.org/source/openssl-1.1.1.tar.gz",
+                    "sha256": "2836875a0f89c03d0fdf483941512613a50cfb421d6fd94b9f41d7279d586a3d"
+                }
+            ]
+        },
+        {
+            "name": "miniupnpc",
+            "no-autogen": true,
+            "make-install-args": ["INSTALLPREFIX=/app"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://miniupnp.free.fr/files/miniupnpc-2.1.tar.gz",
+                    "sha256": "e19fb5e01ea5a707e2a8cb96f537fbd9f3a913d53d804a3265e3aeab3d2064c6"
+                }
+            ]
+        },
+        {
+            "name": "i2pd",
+            "buildsystem": "qmake",
+            "subdir": "qt/i2pd_qt",
+            "make-args": ["USE_UPNP=yes"],
+            "config-opts": ["QMAKE_LIBDIR=/app/lib"],
+            "no-make-install": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/PurpleI2P/i2pd/archive/2.21.1.zip",
+                    "sha256": "c77ff7d06b95b24ad6ad5ccced27ed34da28be88a3f75af26d26e626b5364a8c"
+                },
+                {
+                    "type": "patch",
+                    "path": "i2pd-ldlibs.patch"
+                },
+                {
+                    "type": "file",
+                    "path": "i2pd_qt_wrapper"
+                },
+                {
+                    "type": "file",
+                    "path": "website.i2pd.i2pd.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "website.i2pd.i2pd.desktop"
+                }
+            ],
+            "post-install": [
+                "install -Dm777 i2pd_qt /app/bin/i2pd_qt",
+                "install -Dm777 ../../i2pd_qt_wrapper /app/bin/i2pd_qt_wrapper",
+                "install -Dm644 ../../contrib/i2pd.conf /app/i2pd/i2pd.conf",
+                "install -Dm644 ../../contrib/tunnels.conf /app/i2pd/tunnels.conf",
+                "cp -r ../../contrib/certificates /app/i2pd/certificates",
+                "install -Dm644 resources/images/icon.png /app/share/icons/hicolor/128x128/apps/website.i2pd.i2pd.png",
+                "install -Dm644 ../../website.i2pd.i2pd.appdata.xml /app/share/applications/website.i2pd.i2pd.appdata.xml",
```

```suggestion
                "install -Dm644 ../../website.i2pd.i2pd.appdata.xml /app/share/metainfo/website.i2pd.i2pd.appdata.xml",
```

@l-n-s 	This is why the build failed

---

arrite now it's fixed